Environment/NoiseLevelObserved/doc/spec.md

```diff
@@ -60,33 +60,21 @@ It allows to convey, roughly, information about the precision of the measurement
     + Attribute type: [Text](https://schema.org/Text)
     + Allowed values: one of (`"0"`, `"1"`, `"2"`)
     + Optional
+    
++ `refPointOfInterest` : A reference to a point of interest associated to this observation.
+    + Attribute type: Reference to an entity of type `PointOfInterest`
+    + Optional    
 
 ### Representing acoustic parameters
 
-The number of acoustic parameters measured can vary.
-As a result this model prescribes the following attributes to convey the referred parameters: 
-
-+ `measurand` : An array of strings containing details (see format below) about *each acoustic parameter* observed.
-    + Attribute type: List of [Text](https://schema.org/Text).
-    + Allowed values: Each element of the array must be a string with the following format
-    (a list of values separated by the `|` character):
-`<measurand>| <observedValue>| <description>`, where:
-        + `measurand` : corresponds to a term defined at
-        [http://www.acoustic-glossary.co.uk/definitions-l.htm](http://www.acoustic-glossary.co.uk/definitions-l.htm).
-        + `observedValue` : corresponds to the value for the measurand as a number expressed in decibels. 
-        + `description` : short description of the measurand.
-        + Examples:
-    `"LAeq | 93.6 | A-weighted, equivalent, sound level"  "LAS | 91.6 | A-weighted, Slow, sound level"
-     "LAeq,d | 65.4 | A-weighted, equivalent, day period, sound level"`
-    + Mandatory
-    
-+ In order to enable a proper management of the *historical evolution* of the different acoustic parameters,
-*for each* element described by the `measurand` array list there *MAY* be an attribute which name *MUST* be exactly equal to the
-measurand name described on the `measurand` array. The structure of such an attribute will be as follows:
-    + Attribute name: Equal to the name of the measurand, for instance `LAeq`.
-    + Attribute type: [Number](https://schema.org/Number)
-    + Attribute value: Exactly equal (same unit of measurement) to the value provided in the `measurand` array.
-    
+The number of acoustic parameters measured can vary. *For each* acoustic measurand there *MUST* be an attribute which name *MUST* be exactly equal to the acoustic measurand name, as follows:
++ Attribute name: Equal to the name of the measurand, for instance `LAeq`, `LAeq,d`. It must correspond to a term defined at [http://www.acoustic-glossary.co.uk/definitions-l.htm](http://www.acoustic-glossary.co.uk/definitions-l.htm).
```

Comment:
my observation is that the naming of measurand is not consistent with air quality where you separate the "generic" measure from the specific measurand using _
also, introducing special characters such as "," may generate issues in the context broker and in other backends.

Reply:
@chicco785 FWIW the "," is allowed in attribute names. However, in the ETSI specification we have disallowed it, so using '_' as separator instead of ',' could make sense, even though we will be breaking the international convention used.

Reply:
@manucarrace `measurand` has been removed from the model as per this PR